server/routes/handlers/book/helpers/validateIssuesNeeds.ts

```diff
@@ -54,43 +54,31 @@ export const issuesNeeds = async (req: Request, res: Response): Promise<void> =>
       )
     }
     if (!mappaLevel) {
-      req.session.errors.push(
+      errors.push(
         makeErrorObject({
           id: 'mappaLevel',
           text: 'MAPPA level',
-          values: { mappaLevel },
```

Comment:
Just checking that you intended to delete this - looks slightly inconsistent.

Reply:
Yes, because the only way it can fail validation is if the value isn't entered; in which case, there's nothing to reflect back to the view